openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -210,6 +216,23 @@ def celebrations(self):
         celebrations = get_celebrations_dict(self.effective_user, self.enrollment_object, self.course, browser_timezone)
         return celebrations
 
+    @property
+    def course_goals(self):
+        if COURSE_GOALS_NUMBER_OF_DAYS_GOALS.is_enabled():
+            course_goals = {
+                'goal_options': [],
+                'selected_goal': None
+            }
+            user_is_enrolled = CourseEnrollment.is_enrolled(self.effective_user, self.course_key)
+            if (user_is_enrolled and ENABLE_COURSE_GOALS.is_enabled(self.course_key)):
```

Comment:
will `ENABLE_COURSE_GOALS` become irrelevant once we are fully switched over to the new Goals feature? (I'm trying to understand the relationship between this flag and `COURSE_GOALS_NUMBER_OF_DAYS_GOALS`)

Reply:
this was accidentally committed from the other PR, removed the commit